### ISSUE TITLE
Backup/restore

### ISSUE THREAD

**Issue by ghost** [ivpn/android-app#334]
# Feature request

## Description

The app settings are stored locally, but there is no option to make a backup and restore them from a file.

## Describe the solution you'd like

Add an option to back up and restore the app configuration.

## Describe alternatives you've considered

None

**Comment by Buddel1** [comment]
A backup/restore option would also be useful for the IVPN Android app.

**Comment by ghost** [comment]
@jurajhilje This has not been completed. Please reopen.

**Comment by Buddel1** [comment]
I agree. As this has not been completed yet, this issue should definitely be reopened.
